features/everything.feature

```diff
@@ -1,67 +1,79 @@
 Feature: A game of connect-4
-Scenario: Starting the game and quitting
-   Given starting main.py
-   When the user selects Quit
-   Then the game closes itself
-
-Scenario: Starting the game and choosing 'Play Game'
-   Given starting main.py
-   When the user selects "Play Game"
-   Then the user gets transferred to the next menu
-
-Scenario: Going back to the start menu
-   Given starting main.py
-   When the user is in the gamemode menu
-   When the user selects the <- back button
-   Then the user gets transferred to the start menu
-
-Scenario: Selecting 'Player vs Player'
+#Scenario: Starting the game and quitting
```

Comment:
Pls include finished tests

Reply:
@Surpryze I think Flo meant that you should not comment out finished tests. Please remove the # from finished scenarios when you push the code :) Please change it.

Reply:
ok